examples/java/pom.xml

```diff
@@ -138,12 +137,17 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
+                  <includes>
+                    <include>WordCountIT.java</include>
+                    <include>WindowedWordCountIT.java</include>
+                  </includes>
                   <parallel>all</parallel>
                   <threadCount>4</threadCount>
                   <systemPropertyVariables>
                     <beamTestPipelineOptions>
                       [
                       "--project=apache-beam-testing",
+                      "--tempLocation=gs://temp-storage-for-end-to-end-tests",
```

Comment:
"--tempLocation" is required for BigQuery I/O.

Test runners have ability to build a temp location based on "--tempRoot", like [this in TestDataflowRunner](https://github.com/apache/incubator-beam/blob/master/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/testing/TestDataflowRunner.java#L81). But DirecRunner doesn't. So need to specify it separately. 

Not sure why we want a new temp directory parameter in test options. If it's duplicated, I'd like to refactor in another PR.